.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+fail_fast: true
+repos:
+-   repo: https://github.com/psf/black
+    rev: 22.10.0
```

Comment:
Did you compare the version to the one used in the CI. maybe update both to the highest available one. Check if there is a note in the docs about it as well

Reply:
Let me confirm this. But I believe I did use the versions set on CI

Reply:
Done